modules/claims_api/spec/factories/auto_established_claims.rb

```diff
@@ -175,4 +177,11 @@
     phase_type { 'Preparation for Decision' }
     phase_type_change_ind { '45' }
   end
+  factory :bnft_claim_lc_status_eight, class: OpenStruct do
+    max_est_claim_complete_dt { Faker::Time.backward(days: 5, period: :morning) }
+    min_est_claim_complete_dt { Faker::Time.backward(days: 7, period: :morning) }
+    phase_chngd_dt { Faker::Time.backward(days: 6, period: :morning) }
+    phase_type { 'Complete' }
+    phase_type_change_ind { '78' }
+  end
```

Comment:
I feel like we're getting excessive with these. let's loop back to this at a later date but I would recommend looking at adding traits in the near future to reduce duplication. 
https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#traits